packages/deployment/upgrade-test/Dockerfile

```diff
@@ -87,18 +97,49 @@ RUN chmod +x ./upgrade-test-scripts/*.sh
 SHELL ["/bin/bash", "-c"]
 RUN . ./upgrade-test-scripts/start_to_to.sh
 
-ARG DEST_IMAGE
+# TEST block
 #this is agoric-upgrade-11 / vaults+1
-FROM ${DEST_IMAGE} as agoric-upgrade-11
+ARG DEST_IMAGE
+FROM ghcr.io/agoric/agoric-sdk:36 as agoric-upgrade-11
 ARG BOOTSTRAP_MODE
 ENV THIS_NAME=agoric-upgrade-11 BOOTSTRAP_MODE=${BOOTSTRAP_MODE}
-# this boot doesn't need an upgrade
-
+# start-chain boilerplate
 WORKDIR /usr/src/agoric-sdk/
 COPY ./bash_entrypoint.sh ./env_setup.sh ./start_to_to.sh ./upgrade-test-scripts/
 COPY ./${THIS_NAME} ./upgrade-test-scripts/${THIS_NAME}/
 COPY --from=agoric-upgrade-10-to-11 /root/.agoric /root/.agoric
+RUN chmod +x ./upgrade-test-scripts/*.sh
+SHELL ["/bin/bash", "-c"]
+RUN . ./upgrade-test-scripts/start_to_to.sh
+
+# UPGRADE block
+ARG DEST_IMAGE
+FROM ghcr.io/agoric/agoric-sdk:36 as agoric-upgrade-11-to-12
```

Comment:
Shouldn't this use `DEST_IMAGE` ?

Reply:
Oh my bad, I talked to @michaelfig and he explained to me that this is just running the governance proposal for the upgrade, which does need to run in the previous version. Maybe a comment to that effect would help future readers?